lib/iris/util.py

```diff
@@ -46,6 +46,13 @@ def broadcast_to_shape(array, shape, dim_map):
         the index in *shape* which the dimension of *array* corresponds
         to, so the first element of *dim_map* gives the index of *shape*
         that corresponds to the first dimension of *array* etc.
+    chunks : :class:`tuple`, optional
```

Comment:
Sorry for the delay, there was some confusion on my end about if we were asking for Pytest rather than Unit test, but we are grandfathering in everything made before last month so no changes needed on the testing front. The only thing I would ask is that this docstring could do with referencing the Dask docs for broadcast_to where this appears to be inspired from stating that this is the standard behaviour according to dask and not new functionality that we have decided. Thanks

Reply:
Thank you for reviewing! I added a link in [f12a80c](https://github.com/SciTools/iris/pull/5620/commits/f12a80cafb319e96cef0b47b8b39001c2be1bbd7). Does it look better now?